testData/dbData/views.json

```diff
@@ -3,13 +3,13 @@
         "_id": "_design/views",
         "views": {
             "findPrefsSafeByGpiiKey": {
-                "map": "function(doc) {if (doc.type === 'gpiiKey') emit(doc._id, {'_id': doc.prefsSafeId, 'gpiiKey': doc})}"
+                "map": "function(doc) {\n  if (doc.type === 'gpiiKey') {\nvar record = { gpiiKey: doc};\nif (doc.prefsSafeId) { record._id = doc.prefsSafeId; };\nemit(doc._id, record);\n}\n}"
```

Comment:
It seems adding "\n" decreases the code readability here. Does it make these functions easier to read when functions are loaded into CouchDB? Wouldn't indentations still be missing in CouchDB?

Reply:
@cindyli, I cherry-picked this from @the-t-in-rtf's [GPII-3619 branch](https://github.com/the-t-in-rtf/universal/blob/GPII-3619/testData/dbData/views.json).  While I agree that the "\n" is less readable, I feel it's up to Tony to make a decision about it.

Reply:
We really need something to [provision views from real functions](https://issues.gpii.net/browse/GPII-3301).  In the mean time I tend to use carriage returns out of habit, otherwise a single inline comment can disable your whole function.